### DISCUSSION TITLE
Kubernetes

### DISCUSSION THREAD

**Discussion by ericksg** [saleor/saleor#8907] [Deployment]
Hi, I'm having trouble deploying saleor on Kubernetes, pods continually restarting with almost no concurrency. could someone guide me on this deploy

**Comment by karolkielecki** [comment]
Hi @ericksg. Maybe it's not a guide, but you can find some helpful information regarding Kubernetes [here](https://spectrum.chat/saleor/general/deploy-saleor-in-kubernetes-did-you-try-to-do-that~921ca369-1c2c-4cbb-9b57-9d6cb67c3b3f).
